group_vars/prosody_qa/vars.yml

```diff
@@ -4,7 +4,9 @@
 ---
 
 # email prefix for dev account emails
-email_prefix: '[PPA S&co] '
+email_prefix: '[PPA QA] '
+
+passenger_server_name: "test-ppa.cdh.princeton.edu"
```

Comment:
where are these passenger server names coming from?

Reply:
not sure I understand what you mean? it goes in the passenger config to identify the server

possibly this is redundant and set elsewhere that we could reuse, though

Reply:
I see. So we're using a separate server entirely for passenger, i.e. separate from cdh-test-prosody1 or equivalent ?

Reply:
oh, I understand the question now! Yes, public names / aliases are different from the servers where the code is actually running.